### PR TITLE
Polish solution for all QP problems

### PR DESCRIPTION
**Pull request by stelmo**
Added a setting to make the solver try to get an accurate solution once it has found an approximate one

### CODE REVIEW

**Comment by cylon-x** [comment]
:red_square: &nbsp;Commit 80ad49a4 requires formatting!

Required formatting changes summary:
```
 test/analysis/minimize_metabolic_adjustment.jl | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)
```

**Comment by stelmo** [comment]
Closes #485 if successful

File: test/analysis/minimize_metabolic_adjustment.jl [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
         OSQP.Optimizer;
-        modifications = [silence, change_constraint("CYTBD"; lb = 0.0, ub = 0.0)],
+        modifications = [
+            silence,
+            change_optimizer_attribute("polish", true),
+            change_constraint("CYTBD"; lb = 0.0, ub = 0.0),
+        ],
     )
 
     @test isapprox(moma["BIOMASS_Ecoli_core_w_GAM"], 0.06214149238730545, atol = 0.05)
```
Comment (exaexa) [inline test/analysis/minimize_metabolic_adjustment.jl:21]: pls lower the tolerance back to the original value (TEST_TOLERANCE) to see if it really helped

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit bb33214a is formatted properly.

File: test/analysis/minimize_metabolic_adjustment.jl (hunk 1 above)
Comment (exaexa) [reply test/analysis/minimize_metabolic_adjustment.jl:21]: (you can just `git revert 68d8939`)

**Review by exaexa — CHANGES_REQUESTED** [review]

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 73fa70ed is formatted properly.

**Comment by exaexa** [comment]
looks like we should do with value=0.06, tol=0.01 or something.. :D

**Comment by exaexa** [comment]
but yeah the result with "polish" is much closer. :D

**Comment by stelmo** [comment]
Sigh, OSQP is the worst. Will make a tolerance just for it

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 519fe234 is formatted properly.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit c68b377a is formatted properly.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 2f21708c is formatted properly.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit f8eea9d1 is formatted properly.

**Review by exaexa — APPROVED** [review]

**Pull request merged**